File: replace_cards.js
const fs = require('fs');
const file = 'app/src/main/java/com/example/ui/SettingsScreen.kt';
let text = fs.readFileSync(file, 'utf8');

text = text.replace(/colors = CardDefaults\.cardColors\(containerColor = MaterialTheme\.colorScheme\.surface\),\s*shape = RoundedCornerShape\(12\.dp\)/g, 
  'modifier = Modifier.fillMaxWidth().glassmorphicBackground(MaterialTheme.colorScheme.primary),\n            colors = CardDefaults.cardColors(containerColor = androidx.compose.ui.graphics.Color.Transparent),\n            shape = RoundedCornerShape(12.dp)');

text = text.replace(/colors = CardDefaults\.cardColors\(containerColor = MaterialTheme\.colorScheme\.surfaceVariant\.copy\((.*?)\)\),\s*shape = RoundedCornerShape\(12\.dp\)/g, 
  'modifier = Modifier.fillMaxWidth().glassmorphicBackground(MaterialTheme.colorScheme.primary, 0.08f),\n            colors = CardDefaults.cardColors(containerColor = androidx.compose.ui.graphics.Color.Transparent),\n            shape = RoundedCornerShape(12.dp)');

fs.writeFileSync(file, text);

File: replace_cards.py
import os
import re

file_path = "app/src/main/java/com/example/ui/SettingsScreen.kt"
with open(file_path, "r") as f:
    text = f.read()

pattern = r'colors\s*=\s*CardDefaults\.cardColors\(containerColor\s*=\s*MaterialTheme\.colorScheme\.surface\),\s*\n\s*shape\s*=\s*RoundedCornerShape\(12\.dp\)'
replacement = r"""modifier = Modifier.fillMaxWidth().glassmorphicBackground(MaterialTheme.colorScheme.primary),
            colors = CardDefaults.cardColors(containerColor = androidx.compose.ui.graphics.Color.Transparent),
            shape = RoundedCornerShape(12.dp)"""

text = re.sub(pattern, replacement, text)

with open(file_path, "w") as f:
    f.write(text)
